frontend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:18
+
+WORKDIR /usr/src/app
+
+# Install necessary build dependencies
+RUN apt-get update && apt-get install -y build-essential
+RUN apt-get install python -y
```

Comment:
do we need python here?

Reply:
I think i added this because it is required for node-sass.
I'll double check of we really need this